src/dotnet/CarbonAware.WebApi/api.yaml

```diff
@@ -0,0 +1,180 @@
+openapi: 3.0.0
+info:
+  description: 'Carbon Aware REST API'
+  version: 0.1.0
+  title: Carbon Aware REST API
+
+paths:
+  '/sci-scores':
```

Comment:
Have we thought through auth flow for these at all? Or are we going to leave those for the future?

Reply:
That's exactly what I was asking, but badly. Thank you!